mito-ai/mito_ai/utils/gemini_utils.py

```diff
@@ -227,19 +235,22 @@ async def wait_for_fetch() -> None:
 
 def get_gemini_completion_function_params(
         model: str,
-        contents: str,
+        contents: Sequence[Content],
         message_type: MessageType,
         response_format_info: Optional[Any] = None,
+        system_prompt: Optional[str] = None,
```

Comment:
It doesn't look like we are ever providing the `system_prompt` param right now... but once we update the `convert_messages_for_gemini` function to handle the system messages properly, then we will probably want to provide the system message here.